### PR TITLE
Add ECS mappings generator

### LINKED ISSUE #6: Create vulnerabilities index mapping
## Description

We're working on a new vulnerability detector module (see https://github.com/wazuh/wazuh/issues/14153) which will upload all the vulnerabilities it finds into `wazuh-indexer`.

The module will create an index and set it up with a mappings' template.

We want to adopt the ECS fields as our schema for this index, so it must contain ECS valid documents only. We will support the following fields in the vulnerability index:

- ecs
- base
- event
- agent
- package
- operating system
- vulnerability

The description for each field is described in https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html and the mappings' definition for each field is in the repository https://github.com/elastic/ecs

The documents must be indexed by an unique key. The initial proposal is to create one based on:

`env-name + worker-name + agent-id +  cve-id`

But we might change this when testing the scalability of the index.

## Implementation restrictions

- [x] We must adapt the mappings to work correctly on `wazuh-indexer`
- [x] Mappings compliance must be mandatory

### PR DESCRIPTION
**Pull request by AlexRuiz7**
### Description

This PR adds a script to generate ECS compatible index templates for new Wazuh modules. There is also a README for documentation and the input files for the first module: vulnerability-detector.

### Issues Resolved
Closes #6 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [x] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff
- [ ] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: ecs/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+This script generates the ECS mappings for the Wazuh indices.
+
+### Requirements
+
+- ECS repository clone. The script is meant to be launched from the root level of that repository.
+- Python 3.6 or higher
+- jq
+
+### Folder structrue
```
Comment (asteriscos) [inline ecs/README.md:11]: Did you mean "structure"?

Comment (AlexRuiz7) [reply ecs/README.md:11]: Yes

**Review by asteriscos — APPROVED** [review]
LGTM :heavy_check_mark:

**Review by yenienserrano — APPROVED** [review]
LGTM

**Pull request merged**